packages/babel-plugin-transform-modules-amd/test/fixtures/amd/export-from-2/output.js

```diff
@@ -4,10 +4,13 @@ define(["exports", "foo"], function (_exports, _foo) {
   Object.defineProperty(_exports, "__esModule", {
     value: true
   });
-  Object.defineProperty(_exports, "foo", {
-    enumerable: true,
-    get: function () {
-      return _foo.foo;
-    }
-  });
+
+  function _export(key, get) {
+    Object.defineProperty(_exports, key, {
+      enumerable: true,
+      get
+    });
+  }
+
+  _export("foo", () => _foo.foo);
```

Comment:
Can arrow function be used here? Or does it need to be `function`? If so, can it be anonymous, or does it need to be named `get` like in the original output?

Reply:
Arrow functions were introduced in ES6, I would follow the same solution as for https://github.com/babel/babel/pull/13649/files?w=1#r684666526.

Reply:
Changed to `function()` expression in  9d10002